### PR TITLE
Console app

### PR DESCRIPTION
**Pull request by see-why**
-  Console app behaves in the same way as this example:
    [UI example](https://www.youtube.com/watch?v=vkkgrhD6aXQ)
- Created a `main.rb` file that serves as your console app entry-point. It has the following methods:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- In `main.rb` define the entry point, there is a method called `main` that is invoked at the end of the file. The method does the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+      end
+    end
+    puts "\n"
+  end
+
+  def display_books
+    puts 'BOOKS:'
+    @book_list.each_with_index do |book, i|
+      puts "(#{i}) Title: #{book.title}, Author: #{book.author}"
+    end
```
Comment (Dipeshtwis) [inline main.rb]: - [ ] Everything is working fine but When I tried to create a rental **second** time, I got the error of undefined method 'title'. which I have attached below.

Screenshot from 2021-12-17 18-38-15

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @see-why, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (see-why) [reply main.rb]: Thanks a lot for spotting the error I was saving the rentals into the wrong List.

It should work perfectly now.
🙇

**Review by mikethreels — APPROVED** [review]
Hi Cy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**